### PR TITLE
WIP - but lots of tidy up

### PR DESCRIPTION
**Pull request by pmcb55**
WIP - but lots of Java-related tidy, added glossary vocabs (that use SKOS-XL, which the latest AG supports).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,12 @@ configuration files contained in this repository.
 
 ## Unreleased
 
--
+- Updated glossary vocabs, and added an initial Solid one (based on
+  [SolidProject](https://docs.inrupt.com/developer-tools/javascript/client-libraries/reference/glossary/)).
+- Removed unecessary vocab descriptions from YAMLS (as descriptions should
```
Comment (andydavison) [inline CHANGELOG.md:10]:
```suggestion
- Removed unnecessary vocab descriptions from YAMLS (as descriptions should
```

File: inrupt-rdf/Core/CopyOfVocab/inrupt-meta-server-creational.ttl [hunk 2]
```diff
@@ -9,15 +9,18 @@ prefix schema:   <http://schema.org/>
...
+# Describe our vocabulary - i.e., an English description, its version, who created it, its
+# suggested prefix, its license, etc.
+#
 inrupt_meta_server_creational: a owl:Ontology, inrupt_gen:Ontology ;
 	owl:versionInfo "0.0.1" ;
     owl:versionIRI <https://inrupt.com/vocab/meta/server_creational/0.0.1> ;
  	dcterms:title "LIT Metadata Server Creational Vocabulary"@en ;
-	dcterms:description """The LIT vocabulary providing terms (e.g. classes, properties and text strings (e.g.
+	dcterms:description """The LIT vocabulary providing terms (e.g., classes, properties and text strings (e.g.
  informative labels or error messages) used by LIT servers at server creation time."""@en ;
```
Comment (andydavison) [inline inrupt-rdf/Core/CopyOfVocab/inrupt-meta-server-creational.ttl:24]: maybe not for this PR, but were we moving away from LIT as an acronym? Or is it still relevant here?

File: inrupt-rdf/Core/CopyOfVocab/inrupt-meta-server-runtime.ttl [hunk 3]
```diff
@@ -9,15 +9,18 @@ prefix schema:   <http://schema.org/>
...
+# Describe our vocabulary - i.e., an English description, its version, who created it, its
+# suggested prefix, its license, etc.
+#
 inrupt_meta_server_runtime: a owl:Ontology, inrupt_gen:Ontology ;
 	owl:versionInfo "0.0.1" ;
 	owl:versionIRI <https://inrupt.com/vocab/meta/server_runtime/0.0.1> ;
  	dcterms:title "LIT Metadata Server runtime Vocabulary"@en ;
-	dcterms:description """The LIT vocabulary providing terms (e.g. classes, properties and text strings (e.g.
+	dcterms:description """The LIT vocabulary providing terms (e.g., classes, properties and text strings (e.g.
  informative labels or error messages) used by LIT servers at server runtime."""@en ;
```
Comment (andydavison) [inline inrupt-rdf/Core/CopyOfVocab/inrupt-meta-server-runtime.ttl:24]: same here re: LIT

**Review by andydavison — APPROVED** [review]
One small spelling fix, otherwise looks good to me

File: inrupt-rdf/Core/CopyOfVocab/inrupt-meta-server-creational.ttl (hunk 2 above)
Comment (pmcb55) [reply inrupt-rdf/Core/CopyOfVocab/inrupt-meta-server-creational.ttl:24]: Yeah, we are moving away - I'll quickly fix that. Thanks!

**Pull request merged**